pkg/registry/discovery/endpointslice/storage/storage.go

```diff
@@ -32,16 +35,15 @@ type REST struct {
 // NewREST returns a RESTStorage object that will work against endpoint slices.
 func NewREST(optsGetter generic.RESTOptionsGetter) (*REST, error) {
 	store := &genericregistry.Store{
-		NewFunc:     func() runtime.Object { return &discovery.EndpointSlice{} },
-		NewListFunc: func() runtime.Object { return &discovery.EndpointSliceList{} },
-		ObjectNameFunc: func(obj runtime.Object) (string, error) {
```

Comment:
this part is fine

Reply:
Just to clarify here, are you saying that removing `ObjectNameFunc` is fine here? As far as I could tell most resources weren't implementing this function and instead relying on the default behavior.